src/client/menu/videomenu.c

```diff
@@ -499,33 +508,58 @@ VID_MenuInit(void)
 
 	if (strcmp(vid_renderer->string, "gl3") == 0 || strcmp(vid_renderer->string, "gles3") == 0)
 	{
-		s_intensity_slider.generic.type = MTYPE_SLIDER;
-		s_intensity_slider.generic.name = "color intensity";
-		s_intensity_slider.generic.x = 0;
-		s_intensity_slider.generic.y = (y += 10);
-		s_intensity_slider.cvar = "gl3_intensity";
-		s_intensity_slider.minvalue = 0.1f;
-		s_intensity_slider.maxvalue = 256.0f;
-
-		s_overbrightbits_slider.generic.type = MTYPE_SLIDER;
-		s_overbrightbits_slider.generic.name = "overbrights";
-		s_overbrightbits_slider.generic.x = 0;
-		s_overbrightbits_slider.generic.y = (y += 10);
-		s_overbrightbits_slider.cvar = "gl3_overbrightbits";
-		s_overbrightbits_slider.minvalue = 0.1f;
-		s_overbrightbits_slider.maxvalue = 256.0f;
+		gl1_colorlight = 0;
```

Comment:
it's a pointer, so this should use NULL instead of 0 (same below in the else case)